Source/WebKit/Scripts/webkit/tests/SerializedTypeInfo.cpp

```diff
@@ -506,7 +512,7 @@ Vector<SerializedEnumInfo> allSerializedEnums()
 #if ENABLE(OPTION_SET_SECOND_VALUE)
             static_cast<uint64_t>(EnumNamespace2::OptionSetEnumType::OptionSetSecondValue),
 #endif
-#if !ENABLE(OPTION_SET_SECOND_VALUE)
+#if !(ENABLE(OPTION_SET_SECOND_VALUE))
```

Comment:
It seems like the parentheses in expression(self) aren't needed.

Reply:
I made it this way because you could conceive a single condition statement containing multiple logical operands, say if:

```cpp
#if ENABLE(FOO) && ENABLE(BAR)
...
#else
...
#endif
```

We want this to show up as `#if !(ENABLE(FOO) && ENABLE(BAR))` and not `#if !ENABLE(FOO) && ENABLE(BAR)` in our output. Without the parentheses, it's a bit clunky because we'd have to differentiate between simple and compound conditions.